contrib/pipelines/secadvisor/README.md

```diff
@@ -2,33 +2,223 @@
 
 Implements the IBM Security Advisor pipeline
 
+The security advisor filters the flow data obtained from skydive, performs a data transformation, and saves the information to an object store in JSON format.
```

Comment:
Rather I would write:

The IBM Security Advisor pipeline extracts flow data from skydive, it enriches flow records with status information, batches flows records together into blocks, encodes these blocks in JSON, and compresses using GZIP and stores into any S3 compatible target.